pkg/config/dashboard/dashboard.go

```diff
@@ -75,25 +78,37 @@ func GetChaosDashboardEnv() (*ChaosDashboardConfig, error) {
 }
 
 // ParsePersistTTLConfig parse PersistTTLConfig to persistTTLConfigParsed.
-func ParsePersistTTLConfig(config *PersistTTLConfig) (*ttlcontroller.TTLconfig, error) {
-	SyncPeriod, err := time.ParseDuration(config.SyncPeriod)
+func ParsePersistTTLConfig(config *PersistTTLConfig) (*ttlcontroller.TTLConfig, error) {
+	syncPeriod, err := time.ParseDuration(config.SyncPeriod)
 	if err != nil {
-		return nil, err
+		return nil, errors.WithStack(err)
```

Comment:
I prefer to use `errors.Wrap(err, "prase configuration sync period")` with the message about what we are doing here.